Formula/k/kubefirst.rb

```diff
@@ -30,15 +30,19 @@ class Kubefirst < Formula
   def install
     ldflags = "-s -w -X github.com/kubefirst/runtime/configs.K1Version=v#{version}"
     system "go", "build", *std_go_args(ldflags: ldflags)
-
-    generate_completions_from_executable(bin/"kubefirst", "completion")
   end
```

Comment:
So just to confirm: shell completions which is being removed here is intentional and will no longer be supported?

Reply:
Please confirm this @k8swrangler, due to the hanging issue we cannot ship completions out of the box. Users will have to run `kubefirst completion` on their own after installing.

Reply:
Thanks, edited my comment to make that clear.

Reply:
Shell commands is one of those things that is almost never used in our app anyways. This cli is more an initial interface to the actual app